.github/workflows/ci.yml

```diff
@@ -95,28 +92,12 @@ jobs:
         with:
           fail_ci_if_error: true
 
-  deploy:
-    name: Publish package to PyPI
+  report-status:
+    name: ${{ github.event_name }}-success
```

Comment:
why not just use `success` here ?

Reply:
I want to differenciate the success for the `pull_request`event, and the one for the `push` event. They need different name if we want to make them mandatory before merging the PR.

Reply:
EDIT: I just tested what would happen if we use just a single check for the CI workflow. Turns out we end up with only one check in the branch settings : 
![image](https://user-images.githubusercontent.com/16255570/95048314-f0143400-06e7-11eb-872c-c51ee050502d.png)
And when activating only that check, we have 2 mandatory checks in the CI workflow (one for push event, one for pull_request event).
![image](https://user-images.githubusercontent.com/16255570/95048570-58fbac00-06e8-11eb-80ac-2db19f7894be.png)
I just check, and the PR can not be merged if one of them is missing. So I think we are good with `success` check, as you suggest. I pushed a new commit to fix it.